vagrant_support/bashrc

```diff
@@ -0,0 +1,11 @@
+# .bashrc
+
+# Source global definitions
+if [ -f /etc/bashrc ]; then
+    . /etc/bashrc
+fi
+
+# User specific aliases and functions
+PS1='\[\e[0;31m\]\u\[\e[m\] \[\e[1;34m\]\w\[\e[m\] $ '
```

Comment:
Is the prompt a strict requirement for this to work? I neither object to it nor wish to propose changes, but I'm curious if there's e.g. ansible or expect or something somewhere in Vagrant that cares here.